tests/custom_md_repo/dummymdrepo.py

```diff
@@ -0,0 +1,38 @@
+import os
+from typing import List, Optional
+from mlserver.repository.repository import (
+    ModelRepository,
+    DEFAULT_MODEL_SETTINGS_FILENAME,
+)
+from mlserver.settings import ModelSettings
+from mlserver.errors import ModelNotFound
+
+from ..fixtures import SumModel, ErrorModel, SimpleModel
+
+
+class DummyModelRepository(ModelRepository):
+    def __init__(self, extra_params: Optional[dict] = None) -> None:
```

Comment:
To make it more of an "alternative reference implementation", should we just unpack the `files` kwarg directly here? As in,

```python
def __init__(self, root: str, files: List[str]):
```

Reply:
Done